runtime-config-linux.md

```diff
@@ -144,6 +144,12 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
 
+There are a few facets to this:
+
+* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
```

Comment:
On Wed, Dec 16, 2015 at 10:23:07AM -0800, Vish Kannan wrote:
> > +* Managing the existence of cgroups within the hierarchy (`cgroupsPath`).
> 
> within *what* hierarchy?  `cgroupsPath` is the path to a cgroups
> hierarchy.

This spec requires a unified cgroups hierarchy [1].  cgroupsPath
locates the cgroup for this process within that hierarchy.  The
runtime creates cgroups as needed to make that happen, and removes any
cgroups it created (but which are no longer used) when it cleans up
after itself.  That makes the “within the hierarchy” phrasing clear to
me, but if you have an alternative that you prefer I'll switch to it.

[1]: https://github.com/opencontainers/specs/blob/7c174521427d7752f299e62fec335b45fa6edb3e/runtime-config-linux.md#control-groups